test/config/autotls/certmanager/http01/mesh-issuer.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: http01-issuer
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: "http01-key"
+    solvers:
+      - http01:
+         ingress:
+           podTemplate:
+            metadata:
+              annotations:
+                sidecar.istio.io/inject: "true"
```

Comment:
@ZhiminXiang @tcnghia We really need to stop checking Istio annotations into Serving.  We are having a hard enough time eliminating the ones we have.

Reply:
Yes, we need to get rid of Istio annotations from Serving core components.

Specifically for this one, it is a configuration to support Istio Mesh + CertManager Auto TLS E2E tests. We may have to keep it in the serving repo if we still keep running Istio Mesh + CertManager Auto TLS E2E tests within serving. I could move those yamls into `third_party/certmanager` which seems like a better place to put implementation specific yamls.

Reply:
I don't think we want it in the cert-manager configs either, it violates the separation of concerns.  The goal should be that when Istio is put into mesh mode, the rest of the system doesn't have to change to work.